lib/testers/event-generator-tester.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
      * @param {function} method - A test logic.
      * @returns {any} The returned value with the test logic.
      */
-    describe: (typeof describe === "function") ? describe : function(text, method) {
+    describe: (typeof describe === "function") ? describe : /* istanbul ignore next */ function(text, method) {
```

Comment:
I guess this can’t be tested because `describe` is always a function during our test-run, right? Actually you could overwrite `global.describe` before the test and reset it after the test, but I’m not sure if it is worth that effort.